### PR TITLE
Feature/updated app routes

### PR DESCRIPTION
**Pull request by teddywest32**
Updated the routes to be compatible for Laravel 5.3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ Homestead.yaml
 Homestead.json
 .env
 composer.lock
+/.idea/
```
Comment (nWidart) [inline .gitignore]: Please remove this as this should be in your global gitignore file

File: app/Providers/RouteServiceProvider.php [hunk 2]
```diff
@@ -31,6 +31,8 @@ public function boot()
      */
     public function map()
     {
+        $this->mapWebRoutes();
```
Comment (nWidart) [inline app/Providers/RouteServiceProvider.php]: Please comment those, to avoid loading the routes if not needed.

**Comment by teddywest32** [comment]
@nWidart 
Updated as per your comments.  Is there anything more to be updated?

**Comment by nWidart** [comment]
Thank you! Looks good. 😄

**Pull request merged**